src/gfn/containers/replay_buffer.py

```diff
@@ -155,16 +155,15 @@ def _add_objs(self, training_objects: Transitions | Trajectories | tuple[States]
 
         # Add the terminating states to the buffer.
         if self.terminating_states is not None:
-            assert terminating_states is not None
-            self.terminating_states.extend(terminating_states)
+            assert self.terminating_states is not None
```

Comment:
Again, thank you. Ideally I would have written a test to catch this. I've filed an issue on this here https://github.com/GFNOrg/torchgfn/issues/198